tests/elbv2/test_elbv2_probes.py

```diff
@@ -47,3 +49,50 @@ def test_targets_health_count(aws_client):
     response = targets_health_count(tg_names=tg_names)
     assert response['TestTargetGroup1']['healthy'] == desired_response['TestTargetGroup1']['healthy']
     assert response['TestTargetGroup2']['unhealthy'] == desired_response['TestTargetGroup2']['unhealthy']
+
+
+@patch('chaosaws.elbv2.probes.aws_client', autospec=True)
+def test_all_targets_healthy(aws_client):
```

Comment:
Should we have a test for a positive result also? This test ensures we get the right response of "false" for the mock data, but perhaps one also to show that true is expected when all are healthy?

Reply:
no problems, adding